src/main/java/org/takes/rs/RsPrint.java

```diff
@@ -115,41 +115,52 @@ public void print(final OutputStream output) throws IOException {
     }
 
     /**
-     * Print it into output stream.
+     * Print it into output stream in UTF8.
      * @param output Output to print into
      * @throws IOException If fails
      * @since 0.10
      */
     public void printHead(final OutputStream output) throws IOException {
+        this.printHead(new Utf8OutputStreamWriter(output));
```

Comment:
@victornoel ~~I think it's not correct to enforce encoding to `UTF-8`, user may configure encoding via `-Dfile.encoding` JVM argument, so it's better to use `Charset.defaultCharset()` for encoding.~~

Update: it's not related to this PR, because same approach was used previously, so I've submitted a ticket: #871